connman/src/network.c

```diff
@@ -177,16 +177,16 @@ static void dhcp_success(struct connman_network *network,
 		goto err;
 
 	ipconfig_ipv4 = __connman_service_get_ip4config(service);
+	if (!ipconfig_ipv4)
+		return;
```

Comment:
again, just wondering: here = ignored silently, below = debug logging -> intentional?

Reply:
This case is an error that can be silently ignored. AFAIK every service should have at least the ipconfig setup. The debug below is good for debugging cases where the ipconfig has been re-created before the DHCP returns. I've seen some rare cases happening like that and I left the debug for noticing the issue also via network.c.